### ISSUE TITLE
Recommend building project documentation in CumulusCI-Process-for-Customers-and-Partners wiki

### ISSUE THREAD

**Issue by jacebryan** [SFDO-Sprint-2019-Detroit/CumulusCI-Process-for-Customers-and-Partners#1]
Hi CumulusCI Process for Customers and Partners Team! Great job at the Detroit Sprint! :)

I wanted to submit an Issue to encourage you to host your project documentation in the [CumulusCI-Process-for-Customers-and-Partners wiki](https://github.com/SFDO-Sprint-2019-Detroit/CumulusCI-Process-for-Customers-and-Partners/wiki).

To learn more about this process take a look at the [About wikis](https://help.github.com/en/articles/about-wikis) Github article.

**Comment by brennanbutlerRAD** [comment]
I am looking at this now, so when there is a wiki there needs to be a home page right, what should that page be. 

I will start working on a google doc to add you to. OH and eventually can we use the sidebar thing I like it. 
![bitmoji](https://render.bitstrips.com/v2/cpanel/a38e717b-9601-4351-b3a3-911b8d2d93c3-403a1a5e-029a-46ee-8d39-16a5d73b5464-v1.png?transparent=1&palette=1&width=246)

**Comment by jacebryan** [comment]
Yes - you will want a landing page (i.e. Home page) for the wiki. The good news is that when you create the first page in your Wiki (see image below) you will create the Home page. :)

![WikiHomePage](https://user-images.githubusercontent.com/25934611/61322732-fca32880-a7d3-11e9-90ac-a15eb252d4b1.jpg)

You will want to keep the home page high level (i.e. just describe the content you've created along with links to the corresponding wiki pages that support process and diagrams.) The idea here is to provide high level value around the resources you are making available to the community. Remember - the README is a high level project overview. The wiki is where the people will go to learn more about your project. Think **Table of Contents** for your wiki home page.

Take a look at this Wiki example from a forked version of Outbound Funds: https://github.com/emilymorse/OutboundFunds/wiki

On that note I'd highly recommend reviewing Github's documentation around wikis: https://help.github.com/en/articles/about-wikis

Here are a few additional resources from Github that should help:

Github Guides https://guides.github.com/ 
Video Guides https://www.youtube.com/githubguides 
Github Help https://help.github.com/en

In the end how you structure the wiki is up to you and I am totally willing to give feedback on your Google doc outline. :)

Also - you need to teach me how to make a Bitmoji. The last one I made didn't look like me at all lol

**Comment by brennanbutlerRAD** [comment]
My little OCD self has just realized when building out the WIKI pages, that they are not in the same order as the list. I have been trying to find a way to make them line up with how the home page is organized (Same Order) and I am not able to. Is there a secret I am not aware of?

**Comment by jacebryan** [comment]
@brennanbutlerRAD - Totally hear ya on that! Take a look at this thread: https://stackoverflow.com/questions/39396362/github-wiki-pages-sort-order

Lots of options here.

Let me know which one you decide to go with!

**Comment by brennanbutlerRAD** [comment]
Thanks, Jace, reviewing it now. I am thinking the sidebar will be the simplest way to go.
